packages/app/src/systems/Account/components/AccountItem/AccountItem.tsx

```diff
@@ -75,6 +77,15 @@ export const AccountItem: AccountItemComponent = ({
           Export Private Key
         </Dropdown.MenuItem>
       ),
+      onToggleHidden && (
+        <Dropdown.MenuItem
+          key="hide"
```

Comment:
should this key contain some info unique to the account?  Without unique account info wouldn't we have duplicate AccountItem components with the same 'hide' key?

Reply:
Dropdown menu is visible only when the user clicks on the 3 dots icon of an account. So it's being rendered only once on the screen, hence no duplicates.